.travis.yml

```diff
@@ -1,8 +1,9 @@
 # Documentation: http://docs.travis-ci.com/user/languages/julia/
 language: julia
 julia:
-  - 1.0
+  - 1.3
 os:
+  - osx
```

Comment:
I would get rid of this. Travis OSX VMs are usually very slow to boot up and run.

Reply:
I haven't noticed that with Omniscape. Either way, FWIW, testing on OSX did seem to reveal a Circuitscape bug of some sort.

Reply:
@ViralBShah could you please run tests on the Mac and see if you're able to get this Travis error? If it doesn't fail on your Mac, I would just get rid of this entry

Reply:
Slow boot up is ok, I guess so long as we get the tests. Do you want me to try this branch or master?

Reply:
@ViralBShah either this branch or master should work since the package should build identically on both (I only changed the formatting in Project.toml, not the deps themselves.)

Reply:
I have all tests passing. I am going to guess that the VM is running out of memory.

Reply:
BTW I didn't see any problem with travis mac bootup times.